pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3817,17 +3817,13 @@
         "4.3.0"
       ],
       "BaselineVersion": "4.3.0",
-      "InboxOn": {
-        "netcoreapp2.0": "4.3.0.0",
-        "net461": "4.2.0.0",
-        "netstandard2.0": "4.2.0.0",
-        "uap10.1": "4.3.0.0"
-      },
+      "InboxOn": {},
```

Comment:
I've made this match the other packages that are excluded from netstandard and windows specific with PNSE netstandard implementations e.g. FileSystem.AccessControl and Pipes.AccessControl.